src/ducks/billsMatching/testUtils.js

```diff
@@ -0,0 +1,103 @@
+const parseDate = str => {
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.